src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.cpp

```diff
@@ -117,12 +117,39 @@ void jit_emitter::emitter_preamble(const std::vector<size_t>& in_idxs,
         h->str(Xbyak_aarch64::XReg(preserved_gpr_idxs[i]), pre_ptr(h->sp, -16));
     }
 
+    const size_t aux_vec_idxs_size = aux_vec_idxs.size();
```

Comment:
Please look at this comment https://github.com/openvinotoolkit/openvino/pull/22937#discussion_r1527070032

Reply:
the same behavior as in X64: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.cpp#L149. If I'm wrong then can we connect offline and discuss?

Reply:
was removed from current PR